### PR TITLE
Master => RC

### PR DESCRIPTION
**Pull request by joseph-ross-work**

### CODE REVIEW

**Comment by markdoten** [comment]
Missing `bower.json` version bump

File: src/permalink.js [hunk 1]
```diff
@@ -29,6 +29,10 @@ var Permalink = function () {
     //Check for content permalink
     var content = uriInterpreter.getContentPermalink();
     if (content) {
+        if(content.contentId && content.contentId.indexOf('lb-post') >= 0)  {
```
Comment (markdoten) [inline src/permalink.js]: `if (` -- missing space

Comment (markdoten) [inline src/permalink.js]: Is it possible that `lb-post` could be at the beginning of the string? e.g. `>= -1`

**Comment by markdoten** [comment]
#review

File: src/permalink.js (hunk 1 above)
Comment (joseph-ross-work) [inline src/permalink.js]: -1 is the not found value. Beginning is 0

**Pull request merged**